include/TMU.hpp

```diff
@@ -208,6 +286,30 @@ class TMU {
             .Type = nullptr,
             .Data = (uintptr_t) &thermTemps[3],
         },
+
+        /*
+         * Error Data Links
+         */
+        {
+            .Key = CO_KEY(0x2101, 0, CO_UNSIGNED8 | CO_OBJ___PRW),
```

Comment:
When adding more data links, you only increment the subindex, not the main index. This goes for the other error array data as well
```suggestion
            .Key = CO_KEY(0x2100, 4, CO_UNSIGNED8 | CO_OBJ___PRW),
```
Make sure you update the Mapping links as well